plugins/providers/docker/provider.rb

```diff
@@ -53,6 +45,14 @@ def driver
         @driver
       end
 
+      def usable?(raise_error=false)
```

Comment:
By changing my class method to an instance method, you’ve broken the usability check, because it’s called on the class:

https://github.com/hashicorp/vagrant/blob/v2.2.5/lib/vagrant/environment.rb#L361

https://github.com/hashicorp/vagrant/blob/v2.2.5/lib/vagrant/vagrantfile.rb#L155